Header header # header
uint32 n_bytes # the number of bytes
uint8[] bytes # x coordinates vector
